packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/fixtures/go/lock-asset/chaincode-typescript/src/assetTransfer.ts

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
@jscode017 Is this a verbatim copy of the example contract from fabric-samples or did you modify it in any way?
If it's a verbatim copy we might not need to duplicate it here (need to discuss options if that's the case)

Reply:
@jscode017 Please tag me next time otherwise the notification won't show up that you commented.

> I have added lock in this

Okay, got it.

The separate copies of it at
packages/cactus-plugin-ledger-connector-besu/src/test/solidity/hello-world-contract/
packages/cactus-plugin-odap-hermes/src/test/solidity/lock-asset-contract/

Could be de-duplicated (assuming they are the same) by putting them in the test-tooling package and then importing them from there instead.